tests/conftest.py

```diff
@@ -1327,3 +1331,17 @@ def requests_mock() -> RequestsMockFixture: ...
     # time-machine
     @pytest.fixture  # type: ignore[no-redef]
     def time_machine() -> TimeMachineFixture: ...
+
+
+@pytest.fixture
+def airflow_root_path() -> Path:
+    import airflow
+
+    return Path(airflow.__path__[0]).parent
+
+
+# TODO We should remove it eventually when we fix all tests
+@pytest.fixture
+def require_running_tests_from_sources() -> None:
+    if not (Path(__file__).parents[1] / "airflow" / "__init__.py").exists():
+        pytest.skip("This test requires tests to be run from sources. Skipping it")
```

Comment:
Small nit, could we move this above TYPE_CHECKING block?

Reply:
Sure. I have not noticed there is one :)